coralproject/talk
# Plugin social share

## What does this PR do?
Create a plugin to share comments on **Facebook** and **Twitter**.
Its supposed to fix this #1026.

As I'm using a `react-share` package, It's possible to use other social media and we can think about a way to enable/disable providers.
btw, I'm open to suggestions :)

## How do I test this PR?

- Add the new plugin `talk-plugin-social-share` to your `plugins.json` 
- Run the Talk and check the new share option on embed stream.
- Try to share on Twitter
- Try to share on Facebook (does not work on localhost, just with a online `asset_url`)
